crates/npmrc/src/custom_deserializer.rs

```diff
@@ -127,4 +161,26 @@ mod tests {
         assert_eq!(store_dir, Path::new("/tmp/xdg_data_home/pacquet/store"));
         env::remove_var("XDG_DATA_HOME");
     }
+
+    #[cfg(windows)]
+    #[test]
+    fn test_should_get_the_correct_drive_letter() {
+        let current_dir = Path::new("C:\\Users\\user\\project");
+        let drive_letter = get_drive_letter(current_dir);
+        assert_eq!(drive_letter, Some('C'));
+    }
+
+    #[cfg(windows)]
+    #[test]
+    fn test_default_store_dir_with_windows() {
+        let mocked_current_dir = Path::new("D:\\Users\\user\\project");
+        let mocked_home_dir = Path::new("C:\\Users\\user");
+
+        let store_dir = default_store_dir_windows(mocked_home_dir.to_path_buf(), mocked_current_dir.to_path_buf());
+
+        let current_drive = get_drive_letter(mocked_current_dir);
+        let store_drive = get_drive_letter(&store_dir);
+
+        assert_eq!(current_drive, Some(store_drive.unwrap()));
```

Comment:
Why are you testing just the drive letter?

```suggestion

        let store_dir = default_store_dir_windows(mocked_home_dir.to_path_buf(), mocked_current_dir.to_path_buf());
        assert_eq!(&store_dir, Path::new("D:\\.pacquet-store"));
```

Reply:
I focused the test just in having the correct drive, but yeah, I just changed that